spinalcordtoolbox/utils/fs.py

```diff
@@ -28,6 +29,10 @@ def tmp_create(basename):
     return tmpdir
 
 
+# Taken from https://stackoverflow.com/questions/14693701/how-can-i-remove-the-ansi-escape-sequences-from-a-string-in-python/14693789#14693789
+ansi_escape = re.compile(r'\x1b(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])')
```

Comment:
Side note: The SO answer here contains the following snippet:

```python
# 7-bit C1 ANSI sequences
ansi_escape = re.compile(r'''
    \x1B           # ESC
    (?:            # Case 1. 7-bit C1 Fe escape sequences (except CSI)
        [@-Z\\-_]  #   - Byte in the range 0x40 to 0x5F
    |              # Case 2. Control Sequence Introducer (CSI) escape sequences
        \[         #   - [
        [0-?]*     #   - Parameter bytes
        [ -/]*     #   - Intermediate bytes
        [@-~]      #   - Final byte
    )
''', re.VERBOSE)
```

I didn't realize that regex could be written in this [verbose form](https://www.geeksforgeeks.org/verbose-in-python-regex/), and I like it a lot?